### PR TITLE
grammar

### PR DESCRIPTION
**Pull request by elharo**
@ryanmats some initial cleanup

### CODE REVIEW

File: appengine/wordpress/README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-[App Engine standard environment][appengine-standard] or
+working WordPress project for the
+[App Engine standard environment][appengine-standard] or the
 [App Engine flexible environment][appengine-flexible].
 
 ## Common Prerequisites
 
 * Install [Composer][composer]
-* Create a new Cloud Project on [Developers Console][dev-console]
+* Create a new Cloud Project on the [Cloud Console][cloud-console]
```
Comment (bshaffer) [inline appengine/wordpress/README.md]: how about "Create a new Cloud Project using the [Cloud Console]"

Comment (elharo) [reply appengine/wordpress/README.md]: done

**Comment by elharo** [comment]
Thanks @bshaffer I don't have permission to merge. One of the owners will have to do that.

**Comment by ryanmats** [comment]
thanks @elharo !

**Pull request merged**